### PR TITLE
Fix bug for python<3.4

### PR DESCRIPTION
**Pull request by oeway**
For python2, pathlib needs to be installed in `setup.py`.

asyncio only available since python 3.4, so the requirements needs to be adjust accordingly.

### CODE REVIEW

File: imjoy/__init__.py [hunk 1]
```diff
@@ -3,7 +3,8 @@
 import pathlib
 
 # read version information from file
-HERE = pathlib.Path(__file__).parent
-VERSION_INFO = json.loads((HERE / "VERSION").read_text())
+_here = pathlib.Path(__file__).parent
```
Comment (MartinHjelmare) [inline imjoy/__init__.py]: Use all capital letters for the variable.

**Comment by MartinHjelmare** [comment]
I thought we required Python 3.6 for the engine?

**Comment by oeway** [comment]
Yes for the engine it's python>3.6, but here is about the worker which can be any version.

**Comment by MartinHjelmare** [comment]
Shouldn't we require Python 3.6 if Python version is 3 for the worker too? It will make it harder to keep track of what can be used if we have other Python 3 requirements for the worker than for the engine.

**Comment by oeway** [comment]
> Shouldn't we require Python 3.6 if Python version is 3 for the worker too? It will make it harder to keep track of what can be used if we have other Python 3 requirements for the worker than for the engine.

Limiting the python version used by the plugin worker would be a bit too much for real-life applications, many deep learning projects for example, pin the python version to python 3.5 to work with pytorch.

I think we shouldn't put any constrain if that's not absolutely necessary. It will be harder to keep track indeed if we need to taken care of many different versions, but it's still doable, we just need to make sure all the worker requirements (`"python-socketio", "numpy", "janus"`), and the worker code support all these different version, we should be pretty safe.

**Comment by oeway** [comment]
I will merge this for now since we want to do some manual tests for some important plugins.

**Pull request merged**